nibabel/freesurfer/io.py

```diff
@@ -318,51 +318,81 @@ def read_annot(filepath, orig_ids=False):
         The names of the labels. The length of the list is n_labels.
     """
     with open(filepath, "rb") as fobj:
+        # all data (apart from strings) in an .annot file is stored as
+        # big-endian int32
         dt = ">i4"
+
+        # number of vertices
         vnum = np.fromfile(fobj, dt, 1)[0]
+
+        # vertex ids + annotation values
         data = np.fromfile(fobj, dt, vnum * 2).reshape(vnum, 2)
         labels = data[:, 1]
 
+        # is there a colour table?
         ctab_exists = np.fromfile(fobj, dt, 1)[0]
         if not ctab_exists:
             raise Exception('Color table not found in annotation file')
+
+        # in old-format files, the next field will contain the number of
+        # entries in the colour table. In new-format files, this will be
+        # equal to -2
         n_entries = np.fromfile(fobj, dt, 1)[0]
+
+        # We've got an old-format .annot file.
         if n_entries > 0:
+
+            # orig_tab string length + string
             length = np.fromfile(fobj, dt, 1)[0]
             orig_tab = np.fromfile(fobj, '>c', length)
             orig_tab = orig_tab[:-1]
-
             names = list()
-            ctab = np.zeros((n_entries, 5), np.int)
+            ctab = np.zeros((n_entries, 5), np.int32)
```

Comment:
Any reason not to just use `dt` instead of `np.int32`?

Reply:
No reason, will do